suite-native/notifications/src/components/NotificationRenderer.tsx

```diff
@@ -26,20 +30,28 @@ export const NotificationRenderer = ({ children }: NotificationRendererProps) =>
     const { applyStyle } = useNativeStyles();
     const { top: topSafeAreaInset } = useSafeAreaInsets();
     const transactionNotifications = useSelector(selectOpenedTransactionNotifications);
+    const toastNotifications = useSelector(selectNativeToastNotifications);
 
-    if (!transactionNotifications) return null;
+    if (!transactionNotifications && !toastNotifications) return null;
```

Comment:
```suggestion
    if (!transactionNotifications || !toastNotifications) return null;
```

Reply:
You don't want to render it when either of them fails, right?

Reply:
Also shouldn't this be `A.isNotEmpty(transactionNotifications)`?

Reply:
I need to fix this. It is completely wrong. If this expression would resolve to false, then all the content of the app would be hidden 😅, since it is rendered inside of the JSX as a child.

Reply:
fixed ce38fec3e